status-im/status-react
# [#3881] Stop parsing web3.* as a URL

Fixes #3881 

### Summary:
- Fixes chat message URL parser to not process web3 api calls

### Steps to test:
- Open Status development mode console
- Type web3... commands and verify they are not treated as links

Status: Ready
